src/stko/_internal/optimizers/openmm.py

```diff
@@ -28,7 +31,7 @@ class OpenMMForceField(Optimizer):
 
         You can get all force fields with:
 
-        .. testcode::
+        .. code-block::
```

Comment:
this means its no longer tested -- why?

Reply:
I was trying to avoid installation to build the documentation, but turns out it was installed anyway, so I will try and bring back all these tests. TY